packages/canary-web-components/src/components/ic-data-table/ic-data-table.tsx

```diff
@@ -187,6 +220,12 @@ export class DataTable {
    */
   @Prop() stickyRowHeaders?: boolean = false;
 
+  /**
+   * Sets the method used to truncate longer text in cells where textWrap is `false`.
+   * The `tooltip` truncation pattern allows the overflowing text to be seen in a tooltip. The `showHide` truncation pattern allows the overflowing text to be shown and hidden using the ic-typography "See more"/"See less" buttons.
```

Comment:
```suggestion
   * The `tooltip` truncation pattern allows the overflowing text to be seen in a tooltip. The `show-hide` truncation pattern allows the overflowing text to be shown and hidden using the ic-typography "See more"/"See less" buttons.
```

string should be kebab-case